pkg/model/uimodel.go

```diff
@@ -183,6 +183,9 @@ func (u *UIModel) writeNodeInfo(n *Node, w io.Writer, resources []v1.ResourceNam
 				if !ok {
 					// support computed label values
 					labelValue = n.ComputeLabel(label)
+					if labelValue == label {
+						labelValue = "-"
+					}
```

Comment:
The ComputeLabel method returns the input label name as-is if it doesn't match a custom label; in such cases, it should display `-` instead.

https://github.com/awslabs/eks-node-viewer/blob/3ce0e6f3f536f4043c994dbffdc411b195fbe374/pkg/model/node.go#L294-L305